src/pages/Integrations/List/Page.tsx

```diff
@@ -35,8 +36,12 @@ export const IntegrationsListPage: React.FunctionComponent = () => {
     const integrationRows = useIntegrationRows(integrations);
     const history = useHistory();
 
+    const [activateModal,updateModal] = useState(false);
+    const [model,updateModel] = useState([integrationRows.rows]);
```

Comment:
You probably want to replace the "integrations" variable above with a state, e.g.
```
const [ integrations, setIntegrations ] = useState<Array<Integration>>([
    {
        id: 'foo',
        isEnabled: true,
        name: 'Aha',
        type: IntegrationType.HTTP,
        url: 'https://aha.com'
    },
    {
        id: 'foo-2',
        isEnabled: true,
        name: 'Pager duty',
        type: IntegrationType.HTTP,
        url: 'https://pagerduty.com/weebhook/thatthis'
    }
]);
```
and remove this line, rows will actually read from integration so any change you make it there will reflect on the list.

Reply:
and the `setIntegrations` will be equivalent to your `updateModel`